tests/setup_machine.sh

```diff
@@ -186,6 +186,14 @@ if [ ! -f /etc/sudoers.d/saunafstest ] || ! grep -q '# Client' /etc/sudoers.d/sa
 	END
 fi
 
+if [ ! -f /etc/sudoers.d/saunafstest ] || ! grep -q '# FoundationDB' /etc/sudoers.d/saunafstest >/dev/null; then
```

Comment:
```suggestion
if [ ! -f /etc/sudoers.d/saunafstest ] || ! grep -q '# FoundationDB' /etc/sudoers.d/saunafstest > /dev/null; then
```
Consider adding a space right after the latest `>`

Reply:
I applied it, but please notice that the rest of the script does not use this space.